scripts/check_compliance.py

```diff
@@ -928,8 +929,9 @@ def report_test_results_to_github(suite):
     def set_commit_status(status, msg):
         # 'case' gets set in the loop.
         # pylint: disable=undefined-loop-variable
-        github_commit.create_status(status, name2doc[case.name], msg,
-                                    case.name)
+        github_commit.create_status(status, name2doc[case.name],
+                "{} (build no. {})".format(msg, build_number),
+                case.name)
```

Comment:
Could reformat this as

```python
        github_commit.create_status(
                status, name2doc[case.name],
                "{} (build no. {})".format(msg, build_number),
                case.name)
```

If there are arguments directly after `(`, then the rest of the arguments aren't usually less indented.

Reply:
Done.